src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -68,18 +72,22 @@ class ProfileHeading extends Component {
                         </Typography>
                     </Grid>
                     <Grid item xs={1} md={1} className="rowPadding">
-                        <SchoolIcon height={"24px"} width={"24px"} />
+                        <SchoolIcon style={this.iconScaling()} />
                     </Grid>
                     <Grid item xs={5} md={5} className="rowPadding">
                         <Typography className="rowText">
                             {this.props.user.school}
                         </Typography>
                     </Grid>
-                    <Grid item xs={1} md={1} className="rowPadding">
+                    <Grid item xs={1} md={1} className="rowPadding"
+                     onClick={()=>this.handleEmail()}
+                     style={{cursor:"pointer"}}>
                         <EmailIcon style={this.iconScaling()} />
```

Comment:
Instead, wrap this in an `<a>` to change URL:

    <a href={`mailto:${this.props.user.email}`}>
        <EmailIcon />
    </a>